server/insights/src/main/java/org/infinispan/server/insights/InsightsService.java

```diff
@@ -76,8 +78,8 @@ public void start(BlockingManager blockingManager) {
       try {
          insightsReportController = InsightsReportController.of(insightsLogger, config, insightsReport,
                () -> new InsightsMultiClient(insightsLogger,
-                     new InsightsJdkHttpClient(insightsLogger, config, sslContextSupplier),
-                     new InsightsFileWritingClient(insightsLogger, config)), insightsScheduler,
+                     new InsightsFileWritingClient(insightsLogger, config),
+                     new InsightsJdkHttpClient(insightsLogger, config, sslContextSupplier)), insightsScheduler,
```

Comment:
HttpClient should have precedence to FileWriter as that's preferable solution, even though I think in most use-cases it'll fallback on the FileWriter.

Reply:
OK

Reply:
I revered this change